homeassistant/components/fibaro/quality_scale.yaml

```diff
@@ -0,0 +1,77 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  appropriate-polling:
+    status: exempt
+    comment: |
+      This integration uses a push API. No polling required.
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
```

Comment:
Can we test `test_normalize_url` via the config flow? This way we don't touch internals

Reply:
Agree: Status changed and comment added to quality_scale.yaml